tests/test_matplotlib.py

```diff
@@ -83,3 +83,14 @@ def test_font_manager(selenium_standalone_matplotlib):
             fontlist[list].sort(key=lambda x: x["fname"])
 
     assert fontlist_built == fontlist_vendor
+
+
+@matplotlib_test_decorator
+@run_in_pyodide(packages=["matplotlib"])
+def test_destroy(selenium_standalone_matplotlib):
```

Comment:
Nice, but could you please add more test cases, which check

1. if the expected divs are deleted
2. if calling `plt.close()` multiple time works as expected.
3. if calling `plt.show()` and `plt.close()` multiple times work as expected.